.github/workflows/telosTestNet.deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: Telos TestNet - Deploy Production
+
+on:
+  push:
+    branches:
+      - develop
+      - feature/deploy-with-github-actions
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  SLS_ORG_TOKEN: ${{secrets.SLS_ORG_TOKEN}}
+  chainId: ${{ vars.CHAIN_ID }}
+  sendGridKey: ${{ secrets.SENDGRID_KEY }}
+  twilioAccountSid: ${{ secrets.TWILIO_ACCOUNT_SID }}
+  twilioAuthToken: ${{ secrets.TWILIO_AUTH_TOKEN }}
+  eosHistoryEndpoint: ${{ vars.HISTORY_ENDPOINT }}
+  eosHttpEndpoint: ${{ vars.HTTP_ENDPOINT }}
+  authVerifier: TABLE
+  authContract: logintester1
+  authRetries: 5
+  startAuthTimeout: 1000
```

Comment:
what do these timeout values mean? 1,000 seconds? 1 second?

Reply:
It is I believe in milliseconds. I've just inherited those settings based on what is in production